internal/db/migrations/2024-08-28.0-tss_transactions.sql

```diff
@@ -0,0 +1,35 @@
+-- +migrate Up
+
+CREATE TABLE tss_transactions (
+    transaction_hash TEXT PRIMARY KEY,
+    transaction_xdr TEXT,
+    webhook_url TEXT,
+    current_status TEXT,
+    created_at TIMESTAMPTZ DEFAULT NOW(),
+    updated_at TIMESTAMPTZ DEFAULT NOW(),
+    claimed_until TIMESTAMPTZ
+);
+
+CREATE TABLE tss_transaction_submission_tries (
+    try_transaction_hash TEXT PRIMARY KEY,
+    original_transaction_hash TEXT,
+    try_transaction_xdr TEXT,
+    status INTEGER,
+    updated_at TIMESTAMPTZ DEFAULT NOW()
```

Comment:
Here I guess all columns would be required, right?

Reply:
That's right